desktop/cmp/panel/Panel.ts

```diff
@@ -206,16 +206,17 @@ export const [Panel, panel] = hoistCmp.withFactory<PanelProps>({
                 model: modalSupportModel,
                 item: frame({
                     item,
-                    className: model.isModal ? className : undefined
+                    className: model.isModal ? className : undefined,
+                    testId
                 })
             });
         }
 
         // 4) Return wrapped in resizable affordances if needed, or equivalent layout box
         item =
             resizable || collapsible || showSplitter
-                ? resizeContainer({ref, item, className})
-                : box({ref, item, className, ...layoutProps});
+                ? resizeContainer({ref, item, className, testId})
+                : box({ref, item, className, testId, ...layoutProps});
```

Comment:
Looks like we will end up with the same testId installed twice when modalSupport active - this feature continues to be a bit tricky when it comes to matters of Panel's DOM layout and structure.  Let's review - important though to not allow dupes.